dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Utilities/JsonConverters/DictionaryJsonConverter.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Newtonsoft.Json.Linq;
```

Comment:
Any compatible risk if mix-using `Newtonsoft.Json` and `System.Text.Json`? Or just a redundant namespace using?

Reply:
it's redundant, removed it